server/http_endpoints_test.go

```diff
@@ -823,6 +827,14 @@ func TestEndpointACBufferOptions(t *testing.T) {
 				st.Errorf("want: %d, got: %d", tc.expectedStatus, res.StatusCode)
 			}
 
+			// Prevent diff errors from an unordered list.
```

Comment:
Better create an _additional_ test (e.g. in http_integration_test.go, where several JWT tests already reside) than "hijack" the test named TestEndpointACBufferOptions to check for something very difference.